lib/serverroutes.js

```diff
@@ -21,17 +21,22 @@ const path = require('path')
 const _ = require('lodash')
 const config = require('./config/config')
 const { getHttpPort, getSslPort } = require('./ports')
+const express = require('express')
 
 const defaultSecurityStrategy = './tokensecurity'
 
 module.exports = function(app, saveSecurityConfig, getSecurityConfig) {
   var securityWasEnabled
 
+  app.use(
+    '/admin',
+    express.static(
+      __dirname + '/../node_modules/@signalk/server-admin-ui/public'
+    )
+  )
+
   app.get('/', (req, res) => {
-    res.redirect('@signalk/server-admin-ui')
-  })
-  app.get('/admin', (req, res) => {
-    res.redirect('/@signalk/server-admin-ui')
+    res.redirect('/admin')
```

Comment:
Should we add a redirect from /@signalk/server-admin-ui, reverse of what we had? There may be bookmarks and link here and there.

Reply:
done